i18n/translations/fr/my-forms.json

```diff
@@ -3,11 +3,14 @@
   "nav": {
     "drafts": "Ébauches",
     "published": "Publiés",
-    "all": "Tous"
+    "all": "Tous",
+    "archived": "Archivés"
   },
   "actions": {
     "resumeForm": "Reprendre la rédaction de formulaire en cours",
-    "createNewForm": "Créer un nouveau formulaire"
+    "createNewForm": "Créer un nouveau formulaire",
+    "unarchiveForm": "Désarchiver le formulaire",
+    "duplicatePublishedForm": "Dupliquer le formulaire publié"
```

Comment:
Where does this get duplicated?
 - in what section/folder --> draft or archived? 
 - in what state --> draft or published?